conversions/prefix_conversions.py

```diff
@@ -0,0 +1,111 @@
+"""
+Convert SI and Binary prefixes
+"""
+
+from enum import Enum
+from typing import Union
+
+
+class SI_Unit(Enum):
+
```

Comment:
```suggestion
```
Please lose these blank lines and the blank lines inside your functions.  They force the reader to do a lot of scrolling and they do not help with readability/understandability of the code.

Reply:
when I black the file it adds those two blank lines above class SI_Unit